backend/config.yaml

```diff
@@ -0,0 +1,3 @@
+api:
```

Comment:
What do you think about also having default config so that one can run `edna-server` without thinking about config and still get something meaningful?

Reply:
well, I think we can use default config if a config file is not found or something.